### PR TITLE
Feature 51

### PR DESCRIPTION
**Pull request by AgnieszkaKurek**

### CODE REVIEW

File: functions/exercise6/scriptSpec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals findTheLongestWordWithinTheSentence*/
+
+describe("Finding the longest word within the sentence", function () {
+    it("string 'pig' gives  word 'pig'", function () {
+        var theLongestWord = findTheLongestWordWithinTheSentence("pig");
+        expect(theLongestWord).toEqual("pig");
+    });
+
+    it("string 'pig pig' gives  word 'pig pig'", function () {
```
Comment (Krzysiek102) [inline functions/exercise6/scriptSpec.js]: It should give 'pig', not 'pig pig' - test implementation is ok, this description is not correct.

File: functions/exercise6/script.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+function findTheLongestWordWithinTheSentence(sentence) {
+    var words = sentence.match(/\w[a-z]{0,}/gi);
```
Comment (Krzysiek102) [inline functions/exercise6/script.js]: Do you understand what this line does?
It works, but I have a feeling it is too complicated comparing with what we want to achieve.
But if you can explain it to me, then let's leave it. So please either change it to something simpler or write eplanation what does this line do in details.

File: functions/exercise6/scriptSpec.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    it("sentence 'pink piglet' gives  word 'piglet'", function () {
+        var theLongestWord = findTheLongestWordWithinTheSentence("pink piglet");
+        expect(theLongestWord).toEqual("piglet");
+    });
+    it("sentence 'web development tutorial' gives  word 'development'", function () {
+        var theLongestWord = findTheLongestWordWithinTheSentence("web development tutorial");
+        expect(theLongestWord).toEqual("development");
+    });
+});
```
Comment (Krzysiek102) [inline functions/exercise6/scriptSpec.js:32]: Please add a test definition for empty sentence.

**Review by Krzysiek102 — CHANGES_REQUESTED** [review]
In general ok. There are some minor things worth considering.

**Review by Krzysiek102 — APPROVED** [review]
Ok
